src/ui/control/scale_control.js

```diff
@@ -54,6 +63,16 @@ class ScaleControl {
         this._map.off('move', this._onMove);
         this._map = (undefined: any);
     }
+
+    /**
+     * Set the scale's unit of the distance
+     *
+     * @param {String} unit
```

Comment:
`String` is causing the CI to fail, I believe it should be `string`.

Reply:
Also I think it would be helpful to include the documentation for `unit`, listing out what the supported values are.